src/subscription/SubscriptionPage.jsx

```diff
@@ -71,9 +72,17 @@ export const SubscriptionPage = () => {
 
   return (
     <div className="subscription-page page__payment container-fluid py-5">
-      <FormattedAlertList />
-      {renderContent()}
-      <ConfirmationModal isVisible={isPaymentSuccessful} />
+      <SubscriptionAlerts />
+      {
+      (errorMessageId && errorMessageId === 'empty_subscription')
```

Comment:
Shafqat's BE PR has other error message IDs - https://github.com/edx/subscriptions/pull/48/files
Like `'program_unavailable'`, `'ineligible_program'`, can you add handling for these types?

Reply:
Shafqat said for now just display the `fallback-error` for all other cases unless Michelle gives us proper error strings.

Reply:
Sounds good, thanks Raza!